backend/src/v1/controllers/userController.js

```diff
@@ -7,6 +7,11 @@ const getAllUsers = async (req, res) => {
   res.send({ data: allUsers, error: null });
 };
 
+const getUser = async (req, res) => {
+  const user = await userService.getUser(req.body);
```

Comment:
Pass the username to the `userService` instead. Then you don't have to change the User database file